tck/features/OrderByAcceptance.feature

```diff
@@ -0,0 +1,151 @@
+  Scenario: ORDER BY and ORDER BY DESC should order booleans in the expected orders
+    When executing query: UNWIND [true, false] AS bools RETURN bools ORDER BY bools
+    Then the result should be:
+      | bools |
+      | false |
+      | true  |
+
+    When executing query: UNWIND [true, false] AS bools RETURN bools ORDER BY bools DESC
+    Then the result should be:
+      | bools |
+      | true  |
+      | false |
+
+  Scenario: ORDER BY and ORDER BY DESC should order strings in the expected orders
+    When executing query: UNWIND ['.*', '', ' ', 'one'] AS strings RETURN strings ORDER BY strings
+    Then the result should be:
+      | strings |
+      | ''      |
+      | ' '     |
+      | '.*'    |
+      | 'one'   |
+
+    When executing query: UNWIND ['.*', '', ' ', 'one'] AS strings RETURN strings ORDER BY strings DESC
+    Then the result should be:
+      | strings |
+      | 'one'   |
+      | '.*'    |
+      | ' '     |
+      | ''      |
+
+  Scenario: ORDER BY and ORDER BY DESC should order ints in the expected orders
+    When executing query: UNWIND [1,3,2] AS ints RETURN ints ORDER BY ints
+    Then the result should be:
+      | ints  |
+      | 1     |
+      | 2     |
+      | 3     |
+
+    When executing query: UNWIND [1,3,2] AS ints RETURN ints ORDER BY ints DESC
+    Then the result should be:
+      | ints  |
+      | 3     |
+      | 2     |
+      | 1     |
+
+  Scenario: ORDER BY and ORDER BY DESC should order floats in the expected orders
+    When executing query: UNWIND [1.5,1.3,999.99] AS floats RETURN floats ORDER BY floats
+    Then the result should be:
+      | floats  |
+      | 1.3     |
+      | 1.5     |
+      | 999.99  |
+
+    When executing query: UNWIND [1.5,1.3,999.99] AS floats RETURN floats ORDER BY floats DESC
+    Then the result should be:
+      | floats  |
+      | 999.99  |
+      | 1.5     |
+      | 1.3     |
+
+  Scenario: ORDER BY with LIMIT 0 or a negative parameter should not generate errors
+    And having executed: CREATE (s:Person {name: 'Steven'}), (c:Person {name: 'Craig'})
+
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT 1
+    Then the result should be:
+      | name    |
+      | 'Craig' |
+
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT 0
+    Then the result should be:
+      | name  |
+
+    And parameters are:
+      | limit | -1 |
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT {limit}
+    Then the result should be:
+      | name |
+
+  //This will only work when https://github.com/opencypher/openCypher/pull/25 is merged...
+  Scenario: ORDER BY with a negative LIMIT should fail with a syntax exception
+    And having executed: CREATE (s:Person {name: 'Steven'}), (c:Person {name: 'Craig'})
+    When executing query: MATCH (p:Person) RETURN p.name AS name ORDER BY p.name LIMIT -1
+    Then a SyntaxError should be raised at compile time: [detail]
```

Comment:
That dependency is actually not true. It was really waiting for the implementation of this feature, which is made here: https://github.com/neo4j/neo4j/pull/6735. However, this feature might require the implementation of another `detail` message in `neo4j`, look here: https://github.com/neo4j/neo4j/blob/3.1/community/cypher/compatibility-suite/src/test/scala/cypher/feature/steps/CypherTCKSteps.scala#L104

Of course, the detail must be specified in this file as well.